packages/ui-drilldown/src/Drilldown/index.tsx

```diff
@@ -990,14 +1010,6 @@ class Drilldown extends Component<DrilldownProps, DrilldownState> {
       return null
     }
 
-    if (this.getPageChildById(id)) {
-      warn(
-        false,
-        `Duplicate id: "${id}"! Make sure all options have unique ids, otherwise they won't be rendered.`
-      )
-      return null
-    }
-
```

Comment:
Because of this check and of the double mounting the drilldown did not render any options in `React.StrictMode` since the child with the corresponding `id` would be already in the "cache" and thus this logic would be true.
I moved up the logic for checking duplicate id-ed children to the `renderList` function.